lipx.py

```diff
@@ -79,13 +80,13 @@ def __init__(self, cmd, original_file, modified_file, patch_file):
     def __call__(self):
         ret = False
 
-        print('### Lipx v' + VERSION + ' - Linux IPS Tool ###\n')
+        print(f'### Lipx v{VERSION} - Linux IPS Tool ###\n')
 
         self._setup_files()
 
-        if self.cmd == '-c':
+        if self.cmd == CREATE:
             ret = self.create_ips()
-        elif self.cmd == '-a' or self.cmd == '-ab':
+        elif self.cmd in (APPLY, BACKUP_APPLY, INSPECT):
```

Comment:
i think each argument should have its own function for readability and maintenance reasons, so i would move inspect in a new else with its own inspect()

the only code needed for inspect is the while loop, so it can be moved to its own function, adding an arg flag for inspect and then call this function in apply() and inspect() functions respectively with the flag true or false